### PR TITLE
pre-commit settings added

### PR DESCRIPTION
**Pull request by Dewlar**
configured eslint, prettier and husky(pre-commit and pre-push settings)

### CODE REVIEW

File: src/app/admin/components/editing-user-data/editing-user-data.component.html [hunk 1]
```diff
@@ -2,39 +2,50 @@
...
-      <mat-error *ngIf="passwordValue?.errors?.['required']">Field can't be empty</mat-error>
-      <mat-error *ngIf="passwordValue?.errors?.['minlength']">Length must be at least 10
-        characters. You have {{passwordValue?.value.length}} characters.</mat-error>
+      <input matInput type="password" formControlName="password" />
+      <mat-error *ngIf="passwordValue?.errors?.['required']"
+        >Field can't be empty</mat-error
+      >
+      <mat-error *ngIf="passwordValue?.errors?.['minlength']"
+        >Length must be at least 10 characters. You have
+        {{ passwordValue?.value.length }} characters.</mat-error
```
Comment (burymm) [inline src/app/admin/components/editing-user-data/editing-user-data.component.html]: Could you setup prettier that it format like that 

 ```
<mat-error
        *ngIf="!passwordValue?.errors?.['minlength'] && compareValidator"
        >
    Passwords do not match
</mat-error>
```

File: src/app/admin/components/list-of-users/list-of-users.component.ts [hunk 2]
```diff
@@ -11,35 +11,34 @@ export interface PeriodicElement {
...
-  {position: 3, name: 'Lithium', weight: 6.941, symbol: 'Li'},
-  {position: 4, name: 'Beryllium', weight: 9.0122, symbol: 'Be'},
-  {position: 5, name: 'Boron', weight: 10.811, symbol: 'B'},
-  {position: 6, name: 'Carbon', weight: 12.0107, symbol: 'C'},
-  {position: 7, name: 'Nitrogen', weight: 14.0067, symbol: 'N'},
-  {position: 8, name: 'Oxygen', weight: 15.9994, symbol: 'O'},
-  {position: 9, name: 'Fluorine', weight: 18.9984, symbol: 'F'},
-  {position: 10, name: 'Neon', weight: 20.1797, symbol: 'Ne'},
-];
+// const ELEMENT_DATA: PeriodicElement[] = [
```
Comment (burymm) [inline src/app/admin/components/list-of-users/list-of-users.component.ts:14]: i think need to remove that

Comment (Dewlar) [reply src/app/admin/components/list-of-users/list-of-users.component.ts:14]: I think so too, but in my opinion it should be done by the owner.

File: src/app/admin/components/editing-user-data/editing-user-data.component.html (hunk 1 above)
Comment (Dewlar) [reply src/app/admin/components/editing-user-data/editing-user-data.component.html]: not quite but close
new commit added

File: src/app/admin/components/list-of-users/list-of-users.component.ts (hunk 2 above)
Comment (burymm) [reply src/app/admin/components/list-of-users/list-of-users.component.ts:14]: It comment from u request )

**Review by burymm — APPROVED** [review]

**Comment by ShpakKate** [comment]
did something

**Pull request merged**